.travis.yml

```diff
@@ -2,6 +2,8 @@ language: node_js
 node_js:
 - 6
 - "node"
+env:
+  - NODE_ENV=production
```

Comment:
Since `devDependencies` aren't installed with `NODE_ENV=produciton`, you'll need to adjust the `install` step to include `--production=false` to make sure everything that's needed to run the tests is installed.